### PR TITLE
fix: mf-4679 resolve detail link to support post link

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-4679

### CODE REVIEW

File: packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/NoteCard.tsx [hunk 1]
```diff
@@ -124,8 +125,20 @@ const cardTypeMap = {
     [Type.Share]: CardType.NoteLink,
 } as const
 
-function resolveLenstubeWatchLink(publicationId: string) {
-    return `https://lenstube.xyz/watch/${publicationId}`
+const toHex = (num: number) => {
```
Comment (guanbinrui) [inline packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/NoteCard.tsx:128]: Try this?

```ts
import { toHex } from 'web3-utils'
```

Comment (UncleBill) [reply packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/NoteCard.tsx:128]: Need to pad zero as well.

Comment (guanbinrui) [reply packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/NoteCard.tsx:128]:
```ts
import { padLeft, toHex } from 'web3-utils'
```

Comment (UncleBill) [reply packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/NoteCard.tsx:128]: `toHex` of `web3-util` returns `0xXX`, but we need to pad zero to `XX` to turn into such format `0xXX` or `0x0XXX`.

Besides, is it worth to import such two modules to do such simple thing?

Comment (guanbinrui) [reply packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/NoteCard.tsx:128]: It's widely used.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**